waku/node/waku_node.nim

```diff
@@ -1197,6 +1197,18 @@ proc fetchPeerExchangePeers*(
       error = pxPeersRes.error
     return err(pxPeersRes.error)
 
+proc peerExchangeLoop(node: WakuNode) {.async.} =
+  while node.started:
```

Comment:
nitpick comment:)

We tend to use a `true` in this case. Even though it doesn't sound natural I think we need to keep this technique for consistency purposes ;P
https://github.com/waku-org/nwaku/blob/5c7720c24db807fa31c432c4163f1809cd041e4e/waku/waku_archive/archive.nim#L231